src/content/guides/progressive-web-application.md

```diff
@@ -0,0 +1,150 @@
++       }),
++       new WorkboxPlugin({
++           // these options encourage the ServiceWorkers to get in there fast 
++           // and not allow any straggling "old" SWs to hang around
++           clientsClaim: true,
++           skipWaiting: true,
++       })
+    ],
+    output: {
+      filename: '[name].bundle.js',
+      path: path.resolve(__dirname, 'dist')
+    }
+  };
+```
+
+With that in place, let's see what happens when we do an `npm run build`:
+
+``` bash
+clean-webpack-plugin: /mnt/c/Source/webpack-follow-along/dist has been removed.
+Hash: 6588e31715d9be04be25
+Version: webpack 3.10.0
+Time: 782ms
+                                                Asset       Size  Chunks                    Chunk Names
+                                        app.bundle.js     545 kB    0, 1  [emitted]  [big]  app
+                                      print.bundle.js    2.74 kB       1  [emitted]         print
+                                           index.html  254 bytes          [emitted]
+precache-manifest.b5ca1c555e832d6fbf9462efd29d27eb.js  268 bytes          [emitted]
+                                                sw.js       1 kB          [emitted]
+   [0] ./src/print.js 87 bytes {0} {1} [built]
+   [1] ./src/index.js 477 bytes {0} [built]
+   [3] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [4] (webpack)/buildin/module.js 517 bytes {0} [built]
+    + 1 hidden module
+Child html-webpack-plugin for "index.html":
+     1 asset
+       [2] (webpack)/buildin/global.js 509 bytes {0} [built]
+       [3] (webpack)/buildin/module.js 517 bytes {0} [built]
+        + 2 hidden modules
+```
+
+As you can see, we now have 2 extra files being generated; `sw.js` and the more verbose `precache-manifest.b5ca1c555e832d6fbf9462efd29d27eb.js`. `sw.js` is the Service Worker file and `precache-manifest.b5ca1c555e832d6fbf9462efd29d27eb.js` is a file that `sw.js` imports requires so it can run. Your own generated files will likely be different; but you should have an `sw.js` file there.
+
+So we're now at the happy point of having produced a Service Worker. What to do with it?
+
+## Registering Our Service Worker
+
+In order that our Service Worker can come out and play, we need to register it. We'll do that by adding the registration code below:
+
+__index.js__
+
+``` diff
+  import _ from 'lodash';
+  import printMe from './print.js';
+
++ if ('serviceWorker' in navigator) {
++     window.addEventListener('load', () => {
++         navigator.serviceWorker.register('/sw.js').then(registration => {
++             console.log('SW registered: ', registration);
++         }).catch(registrationError => {
++             console.log('SW registration failed: ', registrationError);
++         });
++     });
++ }
+```
+
+Once more `npm run build` to build a version of the app incuding the registration code. Then serve it with `npm run start`. Navigate to http://localhost:8080 and take a look at the console. Somewhere in there you should see:
+
+```
```

Comment:
> If you didn't already notice...

The missing language here is what's throwing the build error. I know this is a case where it's just console output so something like `bash` or even `js` would probably be fine.